src/antares/model/link.py

```diff
@@ -74,6 +76,15 @@ class LinkProperties(DefaultLinkProperties):
 
 
 class LinkPropertiesLocal(DefaultLinkProperties):
+    hurdles_cost: bool = Field(default=False, alias="hurdles-cost")
```

Comment:
Don't think you should introduce these variables as they are already defined in the DefaultLinkProperties. And you don't use the aliases

Reply:
i removed all these alias. I thought it was useful but no changes when i remove it